created tables in DB corresponding to models and relations

Filari02 opened the pull request:
database: MariaDB

to connect everyone must set these environment parameters (certain values are pinned in discord, do not share them :) )
- spring.datasource.url
- spring.datasource.username
- spring.datasource.password

Filari02 committed:
created entities for database

updated model classes so they could be entities in database

Filari02 committed:
created tables in database and updated entities

Ugniusst commented on Backend/src/main/java/lt/vu/ads/models/Account.java:
```diff
@@ -3,19 +3,30 @@
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(nullable = false)
```
I am not sure, but if ID is auto generated, I think it won't be null.

Filari02 replied:
I used Intellij auto generated code, but it should not impact anything if i delete that @Column row. Will change

Ugniusst commented on Backend/src/main/java/lt/vu/ads/models/Address.java:
```diff
@@ -4,25 +4,33 @@
+    @Column(nullable = false)
     private String street;
 
+    @Column(nullable = false)
```
Maybe Flat number is needed (which is optional)?

Filari02 replied:
you can write flat number in house number. for example, house number - flat number

Foteno commented on Backend/src/main/java/lt/vu/ads/models/Order.java:
```diff
@@ -3,47 +3,67 @@
-    private OrderStatus orderStatus;
-
+    @Column(columnDefinition="BOOLEAN DEFAULT false")
     private boolean isExpress;
```
instead of columnDefinition can just use `private boolean isExpress = false;`

Filari02 committed:
removed columnDefinition

instead of column definition added false values to the boolean fields.

Foteno reviewed (approved):
all good

Filari02 committed:
removed @Column(nullable false) in Idfield

@GeneratedValue(strategy = GenerationType.IDENTITY) automatically creates values so this field will never be null

aule2000 reviewed (approved):
Good
